src/components/Sidebar.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { connect } from "react-redux";
+import { NavLink, withRouter } from "react-router-dom";
+
+import { Badge, Collapse } from "reactstrap";
+import PerfectScrollbar from "react-perfect-scrollbar";
+
+// import { Box } from "react-feather";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+// import { faCircle } from "@fortawesome/free-solid-svg-icons";
+
+import routes from "../routes/index";

```

Comment:
You don't need to write index if you're importing something from the index file.
```javascript
import routes from '../routes/';